packages/mongodb-rag-ingest/src/sources/mongodb-university/makeUniversityPages.ts

```diff
@@ -181,3 +202,26 @@ export function convertVideoTranscriptFromSrtToTxt(transcript: string): string {
     .join(" ")
     .trim();
 }
+
+/**
+  Helper function to create Markdown content for MongoDB University Learning Paths and Courses
+  based on titles, duration, and descriptions.
+ */
+export function generateMarkdown({
```

Comment:
while this function probably captures all the requisite info, i'm reasonably confident that it will be formatted incorrectly. add tests to validate.

can test by seeing if this transforms some test data json like `packages/mongodb-rag-ingest/testData/sampleUniversityCourseWithNestedContent.json` into a specific string.

Reply:
Good call out, I set up a test for this